### PR TITLE
updating nuget specs, references

### PR DESCRIPTION
**Pull request by nayato**

### CODE REVIEW

**Comment by nayato** [comment]
@mtuchkov PTAL

File: host/ProtocolGateway.Host.Common/app.config [hunk 1]
```diff
@@ -30,6 +30,26 @@
...
+        <bindingRedirect oldVersion="0.0.0.0-0.3.1.0" newVersion="0.3.1.0" />
+      </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="DotNetty.Buffers" publicKeyToken="e7a0210a354f294a" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-0.3.1.0" newVersion="0.3.1.0" />
+      </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="DotNetty.Handlers" publicKeyToken="e7a0210a354f294a" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-0.3.1.0" newVersion="0.3.1.0" />
+      </dependentAssembly>
```
Comment (mtuchkov) [inline host/ProtocolGateway.Host.Common/app.config]: DotNetty.Codecs

File: src/ProtocolGateway.Providers.CloudStorage/ProtocolGateway.Providers.CloudStorage.nuspec [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
     <id>Microsoft.Azure.Devices.ProtocolGateway.Providers.CloudStorage</id>
-    <version>1.0.4</version>
+    <version>1.0.6</version>
```
Comment (mtuchkov) [inline src/ProtocolGateway.Providers.CloudStorage/ProtocolGateway.Providers.CloudStorage.nuspec]: Why not 1.0.5?

File: host/ProtocolGateway.Host.Common/app.config (hunk 1 above)
Comment (nayato) [inline host/ProtocolGateway.Host.Common/app.config]: I'll actually ditch the added redirects

**Comment by mtuchkov** [comment]
LGTM

**Pull request merged**